### PR TITLE
[demo] add frontendproxy

### PR DESCRIPTION
**Pull request by puckpuck**
This adds the frontendproxy component to the demo which will handle all user-facing traffic, requiring only a single service to be exposed outside the cluster all UI actions. (The OpenTelemetry collector still needs to be exposed separately to enable browser spans).

frontendproxy is a newer component added to the core demo.

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
Not sure how I accidentally marked this as ready for review but I’ve set it back to draft.

**Comment by puckpuck** [comment]
This PR is ready for review. 😃

**Review by dmitryax — APPROVED** [review]

File: charts/opentelemetry-demo/templates/NOTES.txt [hunk 1]
```diff
@@ -9,33 +9,10 @@
  ╚═════╝    ╚═╝   ╚══════╝╚══════╝    ╚═════╝ ╚══════╝╚═╝     ╚═╝ ╚═════╝
 
 
-- Frontend UI: http://localhost:8080
+- All services are available via the Frontend proxy: http://localhost:8080
```
Comment (TylerHelmuth) [inline charts/opentelemetry-demo/templates/NOTES.txt:12]: Are there specific uris to access each service behind 8080?

Comment (puckpuck) [reply charts/opentelemetry-demo/templates/NOTES.txt:12]: Added all exposed services to this note.

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**